### PR TITLE
Update User Guide

### LINKED ISSUE #124: Update user guide
Mark done: 😃 

- [x] ES
- [x] TK
- [x] Jolyn
- [x] MT
- [x] JH

### PR DESCRIPTION
**Pull request by nmtuan2001**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -158,41 +168,43 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
...
 ### Adding / Editing an interview: `interview`
 Adds an interview to an existing application or edits an existing interview in an application.
 
 Format: `interview INDEX ir/ROUND id/INTERVIEW_DATE it/INTERVIEW_TIME il/LOCATION`
 
 <div markdown="span" class="alert alert-primary">:bulb: <b>Tip:</b><br>
-1. InterviewDate must be specified in the format <em>yyyy-MM-dd</em>.<br>
-2. InterviewTime must be specified in the format <em>HHmm</em>.<br>
+1. INTERVIEW_DATE must be specified in the format <em>yyyy-MM-dd</em>.<br>
+2. INTERVIEW_TIME must be specified in the format <em>HHmm</em>.<br>
```
Comment (teekaytai) [inline docs/UserGuide.md]:
```suggestion
1. `INTERVIEW_DATE` must be specified in the format <em>yyyy-MM-dd</em>.<br>
2. `INTERVIEW_TIME` must be specified in the format <em>HHmm</em>.<br>
```

**Review by teekaytai — COMMENTED** [review]
LGTM! Just one quick suggestion

File: docs/UserGuide.md (hunk 1 above)
Comment (nmtuan2001) [reply docs/UserGuide.md]: Oh I did that at first but this section is with HTML tags, the `` wouldn't work.

Comment (teekaytai) [reply docs/UserGuide.md]: Oh I see, never knew that. Never mind then

Comment (nmtuan2001) [reply docs/UserGuide.md]: I found the fix online actually. Turns out we need to use `<code></code>` instead.

**Pull request merged**